packages/core-typings/src/IRoom.ts

```diff
@@ -148,6 +148,21 @@ export enum OmnichannelSourceType {
 	API = 'api',
 	OTHER = 'other', // catch-all source type
 }
+export interface IVisitorEmailCode {
+	code: string;
+	expire: Date;
+}
+export interface IVisitorServices {
+	emailCode: IVisitorEmailCode[];
```

Comment:
Sorry if I'm missing something here but wouldn't there always be a single verification code at a time? If so, then Y are we storing this as an array and not a simple object?